src/Core/Manifest/ModuleManifest.php

```diff
@@ -225,4 +238,31 @@ public function getModules()
     {
         return $this->modules;
     }
+
+    /**
+     * Get modules sorted by priority
+     *
+     * @return Module[]
+     */
+    public function getSortedModules()
```

Comment:
Let's not have a getModules() and getSortedModules(); Just make getModules() return sorted list?